css-canvas-medical/styles.css

```diff
@@ -0,0 +1,199 @@
+.container {
+  max-width: 900px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.col-full {
+  width: 100%;
+}
+
+a {
+  text-decoration: none;
+}
+
+.light-gray-background {
+  background-color: #fafafa;
+}
+
+.dark-gray-background {
+  background-color: #525351;
+}
+
+.gray-text {
+  color: #888;
+}
+
+.col-one-half {
+  width: 50%;
+}
+
+.padding-50 {
+  padding: 50px;
+}
+
+.col-one-third {
+  flex-basis: calc(100% / 3);
+}
+
+.margin-left-30 {
+  margin-left: 30px;
+}
+
+.align-center {
+  display: flex;
+  align-items: center;
+}
+
+.flex-end {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.blue-button {
+  border: none;
+  border-radius: 5px;
+  padding: 10px 30px;
+  background-color: #294464;
+  color: white;
+}
+
+.contact-button {
+  border: none;
+  border-radius: 5px;
+  padding: 10px 12px;
+  background-color: #294464;
+  color: white;
+}
+
+.margin-top-30 {
+  margin-top: 30px;
+}
+
+.margin-top-20 {
+  margin-top: 20px;
+}
+
+.line-height {
+  line-height: 20px;
+}
+
+.color-white {
+  color: white;
+}
+
+.inline-block {
+  display: inline-block;
+}
+
+.col-one-fourth {
+  flex-basis: calc(100% / 4);
+}
+
+.reduction-30 {
+  width: 30%;
+}
+
+.reduction-15 {
+  width: 15%;
+}
+
+.reduction-70 {
+  width: 60%;
+  font-size: 13px;
+}
+
+.reduction-100 {
+  width: 100%;
+  position: relative;
+  filter: brightness(81%);
+}
```

Comment:
Why are these called `reduction`? It seems like they are serving a purpose that `column` classes would serve.